src/services/users.ts

```diff
@@ -5,6 +5,8 @@ export class Users {
   http: HTTP;
   apiConfig: ApiConfig;
 
+  static $inject = ['apiConfig', 'http'];
```

Comment:
The same here. We use an IoC container to automate dependencies injection. Please check the documentation of the [ioc-lite](https://www.npmjs.com/package/ioc-lite) package.